python: can't open file '/data/XHH/DDPM/Lab1-DDPM/image_diffusion_todo/train.py': [Errno 2] No such file or directory
